src/core/xfa/template.js

```diff
@@ -2053,15 +2088,18 @@ class Font extends XFAObject {
 
   [$toStyle]() {
     const style = toStyle(this, "fill");
-    if (style.color) {
-      if (!style.color.startsWith("#")) {
+    const color = style.background;
+    if (color) {
+      if (color === "#000000") {
+        delete style.background;
```

Comment:
Why do we delete the background if it's black?

Reply:
Because it's the default color in the css:
```css
.xfaFont {
  color: black;
```
so there is no need to have this value.